### PR TITLE
AR-769: Add play and inactive actions. Store played units and don't recommend them.

### PR DESCRIPTION
**Pull request by bbfsdev**
https://issues.kbb1.com/issue/AR-769

Backend already deployed so you are good to go if everything is okay.

### CODE REVIEW

File: src/components/AVPlayer/AVPlayer.js [hunk 1]
```diff
@@ -363,14 +369,15 @@ class AVPlayer extends Component {
   }
 
   onPlay = () => {
-    const { onPlay, item } = this.props;
+    const { onPlay, item, playerPlay } = this.props;
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:372]: Not the best name in player. Maybe rename it to chroniclesPlay or something else?

File: src/components/AVPlayer/AVPlayer.js [hunk 2]
```diff
@@ -879,4 +886,12 @@ class AVPlayer extends Component {
   }
 }
 
-export default withNamespaces()(withMediaProps(withRouter(AVPlayer)));
+export const mapState = () => ({});
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:889]: What do you need this for? Especially exported? Just use {} in connect()

File: src/components/AVPlayer/AVPlayer.js [hunk 3]
```diff
@@ -879,4 +886,12 @@ class AVPlayer extends Component {
...
+
+export const mapDispatch = dispatch => (
```
Comment (gshilin) [inline src/components/AVPlayer/AVPlayer.js:891]: Why to export it? Is it used somewhere else?

**Review by gshilin — CHANGES_REQUESTED** [review]

File: src/components/AVPlayer/AVPlayer.js (hunk 1 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:372]: The event defined inside general redux/module/player.
It is used for recommendation actually not for chronicles.
I would keep the name general, maybe playerPlayAction?

What do you think?

File: src/components/AVPlayer/AVPlayer.js (hunk 2 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:889]: Fixed.

File: src/components/AVPlayer/AVPlayer.js (hunk 3 above)
Comment (bbfsdev) [reply src/components/AVPlayer/AVPlayer.js:889]: Fixed.

**Pull request merged**